graphene/types/tests/test_structures.py

```diff
@@ -19,7 +19,7 @@ def test_list_with_unmounted_type():
 
     assert (
         str(exc_info.value)
-        == "List could not have a mounted String() as inner type. Try with List(String)."
+        == "List can not have a mounted String() as inner type. Try List(String)."
```

Comment:
```suggestion
        == "List cannot have a mounted String() as inner type. Try List(String)."
```

Reply:
I'll defer to the more common usage here. :) `cannot` it is.

Also changing in docstrings in `graphene/pyutils/signature.py`, lines 811 and 818.